test/test_poolmanager.py

```diff
@@ -276,6 +276,14 @@ def test_http_connection_from_host_case_insensitive(self):
         self.assertTrue(pool is other_pool)
         self.assertTrue(all(isinstance(key, PoolKey) for key in p.pools.keys()))
 
+    def test_assert_hostname_and_fingerprint_flag(self):
+        """Assert scheme case is ignored when getting the https key class."""
+        fingerprint = '92:81:FE:85:F7:0C:26:60:EC:D6:B3:BF:93:CF:F9:71:CC:07:7D:0A'
+        p = PoolManager(assert_hostname=True, assert_fingerprint=fingerprint)
+        self.addCleanup(p.clear)
+        p.connection_from_url('https://example.com/')
+        self.assertEqual(1, len(p.pools))
```

Comment:
Can we extend this test to validate that the values are passed properly to the connection pool, and that they can be overridden?